### ISSUE TITLE
Allow to copy from remote tmux to local clipboard

### ISSUE THREAD

**Issue by pszypowicz** [microsoft/vscode-remote-release#8049]
I'm a `tmux` user who copies text from terminal using keyboard. 

In `tmux`, I'm able to enter "copy mode", select text, press "Enter" and selected text will show up in the Clipboard. 

Unfortunately I'm unable to achieve it with Remote SSH extension. However it is working as expected when I'm connect via standard SSH and use `tmux`. 

To be honest I do not know if that's a "Bug" of "Feature Request"

**Comment by bamurtaugh** [comment]
cc @eleanorjboyd I'm making this as a feature request, but if you think it's a bug, please just let me know.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to upvote the issue. If it receives 10 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by Tyriar** [comment]
This is happening because of the way the SSH extension works, VS Code uses SSH to communicate with the remote, but on the remote we launch a regular local shell so the text is copied but it's copied on the remote's clipboard, not the local clipboard.

Given this, I think feature request is right.

**Comment by pszypowicz** [comment]
@bamurtaugh It seems we have 10 upvotes :)

**Comment by kishykumar** [comment]
Can we get this fixed please? It really makes tmux useless on VSCode.

**Comment by kdkavanagh** [comment]
+1, I have remote copy-paste working with tmux on a rhel host via iterm2 on a mac, so I dont think it's a limitation with ssh itself

**Comment by raven42** [comment]
I don't believe this is limited to just Mac vscode. I am running Windows 10 with vscode 1.79.1 with an Ubuntu LTS remote SSH and seeing the same behavior. Copy/paste to the local clipboard works fine for other terminal sessions, but after trying various things found on the web I can't get the copy/paste behavior through tmux to work as expected.

**Comment by zhuoqun-chen** [comment]
Also vote for this feature. 

I configured `remote-ssh` to open `tmux` as default integrated terminal profile. 

After tmux is launched, I can't copy and paste using vim-like `copy-mode-vi` by pressing `y`. I managed to use a tmux-plugin called `tmux-yank` to copy selected words into the command line, but they won't go inside my local macbook's system clipboard. Now I temporarily hold `alt` key to copy via mouse, but being able to yank directly via `y` would be lovely!

Thank you for implementing this feature!
